docs/src/migrate/intellij.md

```diff
@@ -0,0 +1,357 @@
+# How to Migrate from IntelliJ IDEA to Zed
```

Comment:
Was this added by accident? Definitely an interesting doc, just seems unrelated.